src/debug.ts

```
@@ -62,13 +62,21 @@ type MismatchStats = {
 export async function detectMismatches(
-	claudePath?: string,
+	claudePath?: string | string[],
 ): Promise<MismatchStats> {
-	const claudeDir = claudePath ?? path.join(getDefaultClaudePath(), CLAUDE_PROJECTS_DIR_NAME);
-	const files = await glob([USAGE_DATA_GLOB_PATTERN], {
-		cwd: claudeDir,
-		absolute: true,
-	});
+	const claudePaths = claudePath ? (Array.isArray(claudePath) ? claudePath : [claudePath]) : [getDefaultClaudePath()];
+	const allFiles: string[] = [];
+	
```

ESLint

[error] Trailing spaces not allowed.

(style/no-trailing-spaces)